Resources/Prototypes/_CP14/Entities/Clothing/OuterClothing/armor.yml

```
@@ -41,3 +41,27 @@
   - type: ClothingSpeedModifier
     walkModifier: 0.98
     sprintModifier: 0.98
+
+- type: entity
+  parent: CP14ClothingOuterClothingBoneArmor
+  id: CP14ClothingOuterClothingBoneArmorUpgrade
```

```suggestion
  id: CP14ClothingOuterClothingBoneArmorReinforced
```
Стабильность в нейминге